templates/concert-ticket/templates/main.tpl

```diff
@@ -0,0 +1,173 @@
+                                </paragraph>
+                                <paragraph text-align="left" margin="2 0 0 10">
+                                    <text-chunk font="helvetica" font-size="10">{{.TicketNumber}}</text-chunk>
+                                </paragraph>
+                                <image src="qr-code" height="70" width="70" margin="5 0 0 10"></image>
+                            </division>
+                        </table-cell>
+                    </table>
+                </table-cell>
+                <table-cell>
+                    <table columns="2">
+                        {{range .Detail}}
+                        {{template "ticket-detail" dict "FontName" "helvetica" "FontSize" 11 "Name" .FieldName }}
+                        {{template "ticket-detail" dict "FontName" "helvetica" "FontSize" 11 "Name" .FieldValue }}
+                        {{end}}
+                    </table>
+                </table-cell>
+            </table>
+        </division>
+    </table-cell>
+</table>
+<table columns="2" margin="0 0 0 15">
+ <table-cell border-width-top="0.5" border-width-bottom="0.5" border-width-left="0.5" vertical-align="top" indent="0.5" border-style="single">
+    <division margin="0 0 15 15" padding="0, 0, 0, 0">
+        <paragraph text-align="left" margin="10 0 10 0">
+            <text-chunk font="helvetica-bold" font-size="12">Rules of attendance</text-chunk>
+        </paragraph>
+        {{range .RulesOfAttendance}}
+            <paragraph text-align="left" margin="0 0 0 0" line-height="1.3">
+                <text-chunk font="helvetica" font-size="9">{{.}}</text-chunk>
+            </paragraph>
+        {{end}}
+    </division>
+ </table-cell>
+ <table-cell border-width-top="0.5" border-width-bottom="0.5" border-width-right="0.5" vertical-align="top" indent="0" border-style="single">
+    <division margin="0 0 15 15" padding="0, 0, 0, 0">
+        <paragraph text-align="left" margin="10 0 10 0">
+            <text-chunk font="helvetica-bold" font-size="12">Rules of purchase</text-chunk>
+        </paragraph>
+        {{range .RulesOfPurchase}}
+            <paragraph text-align="left" margin="0 0 0 0" line-height="1.3">
+                <text-chunk font="helvetica" font-size="9">{{.}}</text-chunk>
+            </paragraph>
+        {{end}}
+    </division>
+ </table-cell>
+</table>
+
+<table columns="3" margin="10 0 0 10" column-widths="0.5 0.20 0.30">
+    <table-cell vertical-align="top">
+        <division margin="0 20 0 0" padding="0, 0, 0, 0">
+            <paragraph text-align="top" margin="0 0 0 0">
+                <text-chunk font="helvetica-bold" font-size="12">Program/Bands List/Event Description</text-chunk>
+            </paragraph>
+            <paragraph text-align="top" margin="5 0 0 0" line-height="1.3">
+                <text-chunk font="helvetica" font-size="9">A City literally built on Rock n' Roll in Cadott WI, </text-chunk>
```

Comment:
`helvetica` is the default font for text chunks. No need to specify the `font` attribute in this case.